docs/source/development/traffic_portal.rst

```diff
@@ -44,6 +44,32 @@ Installing The Traffic Portal Developer Environment
 #. Navigate to the ``traffic_portal`` subdirectory of your cloned repository.
 #. Run ``npm install`` to install application dependencies into ``traffic_portal/node_modules``. Only needs to be done the first time unless ``traffic_portal/package.json`` changes.
 #. Run ``bower install`` to install client-side dependencies into ``traffic_portal/app/bower_components``. Only needs to be done the first time unless ``traffic_portal/bower.json`` changes.
+#. Before running the next step, make sure that ``compass`` is installed and functioning correctly.
```

Comment:
What if you change 

Before running the next step, make sure that ``compass`` is installed and functioning correctly.

to

Make sure that ``compass`` is installed and functioning correctly by running ``compass version``.  If compass is not running, you can install it on macOSas follows:

Reply:
Done